docker/README.md

```diff
@@ -0,0 +1,13 @@
+# Dockerfile for `celestia-node`
+
```

Comment:
We should also probably add that this is a dockerfile specifically for a `full` node. Maybe we can name the file itself `full.Dockerfile` and make a separate one for the `light` node since we'll need that for devnet.

Reply:
Oh, ok I see) We think the same sometimes ❤️